zabbix/assets/configuration/spec.yaml

```
@@ -0,0 +1,10 @@
+name: zabbix
+files:
+- name: zabbix.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
```

```suggestion
    options:
    - name: zabbix_user
      description: Username to use Zabbix API.
      required: true
      value:
        type: string
```

Let's add the other options included in the conf.yaml.example here. For more information on config specs, see our developer docs https://datadoghq.dev/integrations-core/meta/config-specs/

Once the options are added, you can automatically update the conf.yaml.example file via `ddev -x validate config --sync`